SRM release notes (#3287)
harness/developer-hub: release-notes/service-reliability-management.md

### Fixed issues

- Pagerduty events were not being deleted after the deletion of a project or organization. (SRM-15263)

   This issue has been resolved. Deleting a project or organization no longer leaves behind orphaned Pagerduty events.

- The API endpoint `/cv/api/monitored-service/versions_int1` was not functioning correctly. (SRM-15321)
  
    This issue has been resolved. Now, in cases where the "sources" field is received as null, the system treats it as equivalent to `{ healthSources: [], changeSources: [] }`.

> - The SRM default dashboard is currently non-functional, rendering it unusable. (SRM-15372)

Reviewer: ```suggestion
- The SRM default dashboard was non-functional, rendering it unusable. (SRM-15372)
```
